tests/hadolint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Copyright 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o pipefail
+
+files_to_scan=(
+    "./release_files/Dockerfile.ubuntu"
+    "./release_files/Dockerfile.redhat"
+    "./Dockerfile.ubuntu"
+    "./Dockerfile.redhat"
+)
+
+files_no_proxy_setting=(
+    "./release_files/Dockerfile.ubuntu"
+    "./release_files/Dockerfile.redhat"
+)
+
+docker run --rm -i hadolint/hadolint:latest hadolint -v -V
+
+# TODO: Add allowed registries
```

Comment:
do we want that comment to be merged too?

Reply:
removed